src/backend/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: '@senecacdot/eslint-config-telescope',
+  overrides: [
+    {
+      files: ['src/backend/**/*.js'],
```

Comment:
This `.eslintrc.js` file is already in `src/backend`, we can change this line to

`files: ['./**/*.js'],`